### ISSUE TITLE
Support unsupported message formats that should go into AsyncMessageSequence

### ISSUE THREAD

**Issue by teremterem** [teremterem/AgentForum#65]
The scope of this task depends on which of the tests from #64 fail.
The most likely ones, that will need implementation, are going to be:
1. Detached messages (depends on #54)
2. Error messages (depends on #66)

**Comment by teremterem** [comment]
Setting "Complexity" and "Importance" to "Medium" because most of the work will most likely be done in scope of other tickets.

**Comment by teremterem** [comment]
I think it's already done and I don't see what else to support in scope of this ticket.
